app/src/main/java/com/worldtreeinc/leaves/utility/GetImageFromCamera.java

```diff
@@ -0,0 +1,33 @@
+package com.worldtreeinc.leaves.utility;
+
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.net.Uri;
+import android.os.Bundle;
+import android.provider.MediaStore;
+
+import java.io.ByteArrayOutputStream;
+
+/**
+ * Created by suadahaji on 2/15/16.
+ */
+public class GetImageFromCamera {
```

Comment:
NITPICK - GetImageFromCamera class name is not an ideal for classes in Java. It sounds more like a method name. Invent another name for this class. See the following resources to learn more:
http://cyrille.martraire.com/2006/04/choosing-good-names-for-your-java-classes/
http://www.iwombat.com/standards/JavaStyleGuide.html